### ISSUE TITLE
Minor quest changes

### ISSUE THREAD

**Issue by richardhendricks** [GTNewHorizons/NewHorizons#4194]
Remove ULV tank from LV tank gate

Remove LV Lathe from LV Chemical Reactor gate.

Is LV lathe required? Maybe make it non-main and add an MV lathe quest?

Drain the nether quest mentions lava centrifuging, need to add unlocks at HV.

Check on new EBF hatches - do they require basic ore washer now for byproducts?

**Comment by Bluebine** [comment]
basic ore washer isn't necessary for anything, you can get both of the first 2 byproducts using a centrifuge and a cauldron.
LV lathe isn't needed, and it'd make sense to make it optional.
The tank gating makes sense IMHO- EXCEPT for the super tank, which should be right after the MV pump, since it's really important and useful.

**Comment by richardhendricks** [comment]
LV tank is gated by LV pump.  You think it still should be gated by ULV in steam?

**Comment by Bluebine** [comment]
I guess it wouldn't need to be, and it's probably best to have as few gates as possible

**Comment by shawnbyday** [comment]
Also what about the LV extruder? It is required and gates some tinkers upgrades but aside from that? I just wait until MV to make the extruder and do upgrades at that point. Just a thought.

**Comment by richardhendricks** [comment]
LV extruder can make cells and rubber rings/plates much more efficiently than the alloy smelter or knife. I think it's OK to stay.  It's also useful for the Magnalium crossbow which is a nice upgrade from the wooden crossbow.

**Comment by shawnbyday** [comment]
Yea leave it but was just thinking about the required portion.

**Comment by richardhendricks** [comment]
ULV tank removed from LV tank gate
LV Chemical reactor now gated by H cells. Messy as hell to try moving it around more.
LV lathe is useful for screws and lens, so I'll go ahead and leave it, it also gates diamond spikes and diamond dolly so I think leaving it main is good.  The big problem is just how damn slow certain recipes are, should file a separate ticket to make rods faster in the lathe. That would make it more useful.
HV centrifuge quest added. 
![image](https://user-images.githubusercontent.com/1214811/57197619-249e2200-6f2f-11e9-987b-8d9bb3915a9e.png)
